docs/guides/chip_tool_guide.md

```diff
@@ -243,13 +251,14 @@ This option is available when the Matter device is already present in an IP
 network, but it has not been commissioned to a Matter network yet.
 
 To commission the device, you can use either the setup PIN code or the setup PIN
-code and the discriminator, both of which you obtained in the step 5.
+code and the discriminator, both of which you obtained in the
+[step 5](#step-5-determine-matter-devices-discriminator-and-setup-pin-code).
 Alternatively, you can also use a QR code payload.
 
 ##### Commissioning with setup PIN code
 
-Use the following command pattern to discover devices and try to pair with the
-first discovered one using the provided setup code:
+To discover devices and try to pair with the first discovered device using the
+provided setup code, use the following command pattern:
```

Comment:
That's not actually what this command does anymore.  It will keep trying devices until one of them succeeds or it runs out of things to try, not just try the first one.  Since this PR is trying to not change semantics, maybe a followup issue?

Reply:
Will fix in a separate PR.